oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/NimbusJwtDecoder.java

```diff
@@ -264,6 +270,19 @@ public JwkSetUriJwtDecoderBuilder restOperations(RestOperations restOperations)
 			return this;
 		}
 
+		/**
+		 * Use the given {@link Cache} to store
+		 * <a href="https://tools.ietf.org/html/rfc7517#section-5">JWK Set</a>.
+		 *
+		 * @param jwkSetCache the {@link Cache} to be used to store JWK Set
+		 * @return a {@link JwkSetUriJwtDecoderBuilder} for further configurations
+		 */
```

Comment:
Will you please add a `@since 5.4` annotation here?

Reply:
Thanks for your feedback, makes perfect sence, I'll address your comments as soon as I can